CONTRIBUTING.md

```diff
@@ -196,62 +196,45 @@ You can set the `DEBUG` environment variable to turn on debugging logs. E.g. `DE
+We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, consider whether the problem:
 
-- Whether the problem is really worth addressing
-- Whether the problem can be fixed with a smarter default
-- Whether the problem has workaround using existing options
-- Whether the problem can be addressed with a plugin instead
+- is really worth addressing
+- can be fixed with a smarter default
+- has workaround using existing options
+- can be addressed with a plugin instead
 
-## Docs translation contribution
+## Docs Translation Contribution
 
-If you would like to start a translation in your language, you are welcome to contribute! Please join [the #translations channel in Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
+If you would like to start a translation in your language, you are welcome to contribute! Please join the [#docs channel](https://discord.com/channels/804011606160703521/855049073157341234) in [Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
 
-The english docs are embedded in the main Vite repo, to allow contributors to work on docs, tests and implementation in the same PR. Translations are done by forking the main repo.
+The [English docs are embedded in the main Vite repo](https://github.com/vitejs/vite/tree/main/docs) to enable contributors to work on docs, tests, and implementation in the same PR. Translations of those docs are contained in [separate repos in the `vitejs` org](https://github.com/vitejs?q=docs&type=all&language=&sort=).
 
-### How to start a translation repo
-
-1. In order to get all doc files, you first need to clone this repo in your personal account.
-2. Keep all the files in `docs/` and remove everything else.
-
-   - You should setup your translation site based on all the files in `docs/` folder as a VitePress project.
-     (that said, `package.json` is need).
-
-   - Refresh git history by removing `.git` and then `git init`
-
-3. Translate the docs.
-
-   - During this stage, you may be translating documents and synchronizing updates at the same time, but don't worry about that, it's very common in translation contribution.
-
-4. Push your commits to your GitHub repo. you can setup a netlify preview as well.
-5. Use [Ryu-cho](https://github.com/vuejs-translations/ryu-cho) tool to setup a GitHub Action, automatically track English docs update later.
-
-We recommend talking with others in Vite Land so you find more contributors for your language to share the maintenance work. Once the translation is done, communicate it to the Vite team so the repo can be moved to the official vitejs org in GitHub.
+To add a new language to the Vite docs, see [`vite-docs-template`](https://github.com/tony19/vite-docs-template/blob/main/.github/CONTRIBUTING.md).
```

Comment:
I think the docs template repo is a better home for the translations contributing guide, so I just linked it here.

@bluwy I think I should transfer that repo to the `vitejs` org. Thoughts?